### PR TITLE
feat: redirect user to previous page on login

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

**Review by kmaid — APPROVED** [review]

File: apps/frontend/src/auth/Signin.tsx [hunk 1]
```diff
@@ -6,9 +6,12 @@ const Home: React.FC<{}> = () => {
   const { loginWithRedirect } = useAuth0();
 
   const signin = () => {
-    // TODO pass a redirect_uri with a query param containing the current page;
-    // and send the user back to that original page in handleRedirectCallback
-    return loginWithRedirect({ prompt: 'login' });
+    return loginWithRedirect({
+      prompt: 'login',
+      appState: {
+        targetUrl: window.location.pathname + window.location.search,
```
Comment (jeysal) [inline apps/frontend/src/auth/Signin.tsx:12]: Any reason why this includes only `pathname` and `search`?

**Review by jeysal — CHANGES_REQUESTED** [review]
Test please :upside_down_face: `prompt` is already being tested, so the fundamental test setup is there

File: apps/frontend/src/auth/Signin.tsx (hunk 1 above)
Comment (fampinheiro) [reply apps/frontend/src/auth/Signin.tsx:12]: The redirect URL expects a pathname on the same origin.
https://github.com/yldio/asap-hub/blob/master/apps/frontend/src/auth/AuthProvider.tsx#L16

Comment (jeysal) [reply apps/frontend/src/auth/Signin.tsx:12]: Yeah it seems like an absolute URL would not be supported, but there seems to be no reason why `hash` should be left out

**Pull request merged**